src/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@
 #
 # src/CMakeLists.txt
 #
+set (SUPPL_NAME core)
```

Comment:
SUPPL_NAME can be "core" or "supplements". I think it's NOT a good name for this variable, since SUPPL_NAME is also used in src subdirectories (e.g. src/seis/CMakeLists.txt). It's better to rename SUPPL_NAME and SUPPL_PURPOSE to other names.

Reply:
OK.  Since nothing is really named "core" it is more of a flag. Maybe use SHARED_LIB_PURPOSE for the purpose strings and SHARED_LIB_NAME for the SUPPL_NAME part in src/CMakeLists.txt?

Reply:
The idea is that they are both shared libraries but only on is supplemental, so SUPPL is not a good name I agree.

Reply:
These two names are good for me.